### PR TITLE
[DPC-4026] Setup activerecord-session_store gem

### PR DESCRIPTION
**Pull request by MEspositoE14s**
## 🎫 Ticket

https://jira.cms.gov/browse/DPC-4026

## 🛠 Changes

Added and configured the `activerecord-session_store` gem so that sessions are stored in the DB.

## ℹ️ Context for reviewers

This should allow us to securely store a users IdP auth token in their session.

## ✅ Acceptance Validation

Ran dpc-portal locally and verified that rows were created in the session table when I accessed the site.

## 🔒 Security Implications

- [ ] This PR adds a new software dependency or dependencies.
- [ ] This PR modifies or invalidates one or more of our security controls.
- [ ] This PR stores or transmits data that was not stored or transmitted before.
- [ ] This PR requires additional review of its security implications for other reasons.

If any security implications apply, add Jason Ashbaugh (GitHub username: StewGoin) as a reviewer and do not merge this PR without his approval.

### CODE REVIEW

File: dpc-portal/config/environments/production.rb [hunk 1]
```diff
@@ -120,3 +120,5 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 end
+# Session serializer
+ActiveRecord::SessionStore::Session.serializer = :json
```
Comment (jdettmannnava) [inline dpc-portal/config/environments/production.rb:124]: I don't think we want this here.

File: dpc-portal/config/environments/test.rb [hunk 2]
```diff
@@ -63,3 +63,5 @@
   # Annotate rendered view with file names.
   # config.action_view.annotate_rendered_view_with_filenames = true
 end
+# Session serializer
+ActiveRecord::SessionStore::Session.serializer = :json
```
Comment (jdettmannnava) [inline dpc-portal/config/environments/test.rb:67]: I don't think we want this here

File: dpc-portal/config/environments/production.rb (hunk 1 above)
Comment (MEspositoE14s) [reply dpc-portal/config/environments/production.rb]: Consider it gone.

File: dpc-portal/config/environments/test.rb (hunk 2 above)
Comment (MEspositoE14s) [reply dpc-portal/config/environments/test.rb]: This one, too.

**Review by jdettmannnava — APPROVED** [review]
LGTM!

**Pull request merged**